### ISSUE TITLE
Patch for updating to Kotlin 1.7.20

### ISSUE THREAD

**Issue by Xarno** [Dyescape/kotlin-maven-symbol-processing#67]
Easy Patch to work with Kotlin 1.7.20. Nothing else need to be changed.

```
From ed4d9613b7ae186668445224a48da0fd314fe305 Mon Sep 17 00:00:00 2001
From: Xarno
Date: Thu, 10 Nov 2022 17:05:26 +0100
Subject: [PATCH 1/1] Connect to Kotlin 1.7.20

---
 pom.xml | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/pom.xml b/pom.xml
index 07c4a70..9b6281e 100644
--- a/pom.xml
+++ b/pom.xml
@@ -44,29 +44,30 @@

     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <kotlin.version>1.7.20</kotlin.version>
     </properties>

     <dependencies>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-stdlib-jdk8</artifactId>
-            <version>1.6.21</version>
+            <version>${kotlin.version}</version>
         </dependency>
         <dependency>
             <groupId>com.google.devtools.ksp</groupId>
             <artifactId>symbol-processing-cmdline</artifactId>
-            <version>1.6.21-1.0.5</version>
+            <version>${kotlin.version}-1.0.8</version>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-compiler</artifactId>
-            <version>1.6.21</version>
+            <version>${kotlin.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-maven-plugin</artifactId>
-            <version>1.6.21</version>
+            <version>${kotlin.version}</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
@@ -78,7 +79,7 @@
             <plugin>
                 <groupId>org.jetbrains.kotlin</groupId>
                 <artifactId>kotlin-maven-plugin</artifactId>
-                <version>1.6.21</version>
+                <version>${kotlin.version}</version>

                 <executions>
                     <execution>
@@ -106,7 +107,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>3.3.0</version>
+                <version>3.4.1</version>
                 <executions>
                     <execution>
                         <id>shade</id>
@@ -191,7 +192,7 @@
                     <plugin>
                         <groupId>org.jetbrains.dokka</groupId>
                         <artifactId>dokka-maven-plugin</artifactId>
-                        <version>1.6.21</version>
+                        <version>${kotlin.version}</version>
                         <executions>
                             <execution>
                                 <phase>prepare-package</phase>
--
2.37.1 (Apple Git-137.1)
```

**Comment by onXoot** [comment]
any time to update kotlin version and release new version?

**Comment by MrDienns** [comment]
They there @onXoot, @Xarno

We're currently focusing our efforts on a larger project. We'll pick this up as soon as we can.

**Comment by fxshlein** [comment]
Hi!

Sorry for the long wait - we finally just released version `1.4` for kotlin `1.7.22`.
